client/verta/tests/test_model_registry/test_model_version.py

```diff
@@ -39,7 +39,9 @@ class TestModelVersion:
 
     def test_create(self, registered_model):
         name = verta._internal_utils._utils.generate_default_name()
-        assert registered_model.create_version(name)
+        version = registered_model.create_version(name)
+        assert version
+        print(version)
```

Comment:
The `assert` will help `pytest` generate its test failure message
```suggestion
        assert repr(version)
```

Reply:
Actually this change is not needed because test_repr checks the same thing. Removed it